javascript/javascript1/week1/index.html

```diff
@@ -0,0 +1,87 @@
+        <h2>Age-ify (A future age calculator):</h2>
+        <fieldset>
+            <legend>You can count your future age year here</legend>
+        <label for="birthyear">Year Of Birth:</label>
+        <input type="number" id="birthyear"   min="1950" max="2099" step="1" value="1989">
+        <br>
+        <label>Future Year:</label>
+        <input type="number" id="futureyear"  min="2022" max="2099" step="1" value="2022">
+        <br>
+        <div class="agebutton">
+        <button onclick="futureAge()">Future Age</button>
+        <textarea id="calculatedage" cols="25" rows="2" ></textarea>
+        </div>
+    </fieldset>
+  </div>
+
+  <!-- Goodboy-Oldboy (A dog age calculator) -->
+  <div class="dogagecalculator">
+    <h2>Goodboy-Oldboy (A dog age calculator):</h2>
+    <fieldset>
+        <legend>You can count your dog's future age year here</legend>
+    <label for="dogbirthyear">Dog's Year Of Birth:</label>
+    <input type="number" id="dogbirthyear"   min="1950" max="2099" step="1" value="2017">
+    <br>
+    <label for="dogfutureyear"> Dog's Future Year:</label>
+    <input type="number" id="dogfutureyear"  min="2022" max="2099" step="1" value="2027">
+    <br>
+    <div class="dogagebutton">
+    <button onclick="dogFutureAge()">Dog's Age</button>
+    <textarea id="dogcalculatedage" cols="28" rows="2" ></textarea>
+    <textarea id="dogcalculatedhumanage" cols="28" rows="2" ></textarea>
+    </div>
+</fieldset>
+</div>
+
+<!-- Ez Namey (Startup name generator) Optional -->
+<div class="namegenerator">
+    <h2>Ez Namey (Startup name generator) Optional:</h2>
+    <fieldset>
+        <legend>You can generate Startup name here</legend>
+        <button onclick="startUpNameGenerator()">Click Here</button>
+        <textarea id="uniquename" cols="28" rows="2" ></textarea>
+</fieldset>
+</div>
+
+<!-- Housey pricey (A house price estimator) -->
+<div class="housepricey">
+    <h2>Housey pricey (A house price estimator):</h2>
+    <fieldset>
+        <legend>Estimate your house price here</legend>
+        <label for="width">House width(m):</label>
+        <input type="number" id="width" max="100" step="1" value="8"><br>
+        <label for="depth">House Depth(m):</label>
+        <input type="number" id="depth" max="100" step="1" value="10"><br>
+        <label for="height">House Height(m):</label>
+        <input type="number" id="height" max="100" step="1" value="10"><br>
+        <label for="garden">Garden Size(m2):</label>
+        <input type="number" id="garden" max="1000" step="1" value="100"><br>
+
+        <label for="currentrent">Your Current Rent:</label>
+        <input type="tel" id="currentrent" value="2500000"><br>
+        
+        <div class="housepricey-button">
+            <button onclick="housePriceEstimator()">House Price</button>
+            <textarea id="houseprice" cols="28" rows="2" ></textarea>
+        </div>
+
+       
+</fieldset>
+</div>
+
+  
+   <script src="script.js"></script>
```

Comment:
It would be great if you can name files with meaningful names, imagine you have 20 files, would script1, script2, etc be a good choice? 🙂